tracing/tck/src/main/java/org/eclipse/microprofile/telemetry/tracing/tck/rest/RestClientSpanDefaultTest.java

```diff
@@ -68,7 +68,7 @@ public static WebArchive createDeployment() {
                 .addClasses(InMemorySpanExporter.class, InMemorySpanExporterProvider.class)
                 .addAsLibrary(TestLibraries.AWAITILITY_LIB)
                 .addAsServiceProvider(ConfigurableSpanExporterProvider.class, InMemorySpanExporterProvider.class)
-                .addAsResource(new StringAsset("otel.experimental.sdk.enabled=true\notel.traces.exporter=in-memory"),
+                .addAsResource(new StringAsset("otel.traces.exporter=in-memory"),
```

Comment:
With this setting, the span should not be created, right? MP Telemetry disables the tracing by default. It only works if we set the property to be `true`.

Reply:
That's right. The test makes sure that no spans were made because the property was not set at all. By default, it is off